tests/sentry/digests/test_utilities.py

```diff
@@ -279,5 +420,67 @@ def test_everyone_with_owners(self):
         }
         assert_get_personalized_digests(self.project, digest, expected_result)
 
+    @with_feature("organizations:workflow-engine-ui-links")
+    def test_everyone_with_owners_with_workflow_id(self):
+        events = self.create_events_from_filenames(
+            self.project, ["hello.moz", "goodbye.moz", "hola.moz", "adios.moz"]
+        )
+        records = [
+            event_to_record(event, (self.rule_with_workflow_id,))
+            for event in events + self.team1_events
+        ]
+        digest = build_digest(self.project, sort_records(records))[0]
+        expected_result = {
+            self.user1.id: set(events),
+            self.user2.id: set(events),
+            self.user3.id: set(events + self.team1_events),
+            self.user4.id: set(events),
+            self.user5.id: set(events),
+        }
+        assert_get_personalized_digests(self.project, digest, expected_result)
+
+    @with_feature("organizations:workflow-engine-trigger-actions")
+    def test_everyone_with_owners_with_legacy_rule_id(self):
+        events = self.create_events_from_filenames(
+            self.project, ["hello.moz", "goodbye.moz", "hola.moz", "adios.moz"]
+        )
+        records = [
+            event_to_record(event, (self.rule_with_legacy_rule_id,))
+            for event in events + self.team1_events
+        ]
+        digest = build_digest(self.project, sort_records(records))[0]
+        expected_result = {
+            self.user1.id: set(events),
+            self.user2.id: set(events),
+            self.user3.id: set(events + self.team1_events),
+            self.user4.id: set(events),
+            self.user5.id: set(events),
+        }
+        assert_get_personalized_digests(self.project, digest, expected_result)
+
+    @with_feature("organizations:workflow-engine-trigger-actions")
+    def test_simple_with_workflow_id_flag_off_fallback(self):
```

Comment:
Do we have any tests covering the "belongs to this project" checks for workflow lookups by chance?